app/views/users/_proposal.html.erb

```diff
@@ -7,28 +7,34 @@
 
   <% if proposal.retired? %>
 
-    <td class="text-center">
+    <td class="text-center" colspan="2">
```

Comment:
As I've seen in the screenshots (thanks!) the buttons appear inside the `<td>` aligned to the center and in cases where the element has margin around it looks weird, but what if we use the [`.text-right`](https://github.com/consul/consul/blob/master/app/assets/stylesheets/admin.scss#L64) selector to make it a bit more _clean_? This is just a silly question, you can perfectly discard it 🙃

Reply:
Thanks for the feedback!

I try with the `text-right`but i don't like so much (for the other user I think is better centered). Anyway all the users views needs a good redesign 😅. I add the "proposal retired" to make the label bigger and clarify better to users. 

On the right a little space to the border... 
<img width="1230" alt="right" src="https://user-images.githubusercontent.com/631897/29740825-4571b542-8a60-11e7-8b40-cc5f708db70d.png">

I think is better in the center...
<img width="286" alt="all_users" src="https://user-images.githubusercontent.com/631897/29740831-55fc0160-8a60-11e7-9e26-2a4ef3e100b1.png">

Reply:
So perfect then!